php-classes/RecordsRequestHandler.class.php

```diff
@@ -414,6 +414,11 @@ public static function handleMultiSaveRequest()
 
                 // call template function
                 static::onRecordSaved($Record, $datum);
+                // fire event
+                Emergence\EventBus::fireEvent('afterRecordTransaction', __CLASS__, array(
```

Comment:
Let's use `$className::getRootClass()` as the context. `__CLASS__` in this context would always fire it on RecordsRequestHandler, you'd need to use `get_called_class()` to get the implementation's class name. Let's use the model as the context instead though so these handlers can be alongside the model handlers